### DISCUSSION TITLE
How to extract year and month from date field?

### DISCUSSION THREAD

**Discussion by LukaszTod** [YetiForceCompany/YetiForceCRM#17263] [How to]
How can I extract year and month from date field ("data_oplacenia") and save them do text fields, using workflow editor?

![image](https://github.com/YetiForceCompany/YetiForceCRM/assets/34128804/bdc7cb7a-ab00-4d6e-9451-31307c31a6b2)

**Comment by zielonekrzeslo** [comment]
Najpierw w pliku 
/var/www/yetiforce/modules/com vtiger_ workflow/expression engine/VTExpressionEvaluater.php 
dodaj definicję funkcji i w tym samym pliku do _construct dodaj odwołanie do tej funkcji, a potem w pliku 
/var/www/yetiforce/modules/com vtiger_ workflow/expression engine/VTExpressionManager.php
w expressionFunctions() dodaj nazwę twojej funkcji, aby się wyświetliła na liście dostępnych do wybrania.
